src/App.js

```diff
@@ -1,11 +1,29 @@
-import React from "react";
-import "./index.scss";
+import React, { useEffect } from "react";
+import "./styles/index.scss";
 import Home from "./pages/Home";
+import { Route, Routes, BrowserRouter as Router, Link } from "react-router-dom";
+import EditMovie from "./components/FormikMovieForms/EditMovie";
+import FormikAddMovie from "./components/FormikMovieForms/FormikAddMovie";
+import { useDispatch } from "react-redux";
+import { loadMovies } from "./store/thunk";
+import Error404 from "./components/404/Error404";
 
 const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loadMovies());
+  }, []);
   return (
     <>
-      <Home />
+      <Router>
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/add-movie" element={<FormikAddMovie />} />
+          <Route path="/edit-movie" element={<EditMovie />} />
+
```

Comment:
I'd recommend adding route /movie/{id} for movie details
https://v5.reactrouter.com/web/example/query-parameters

P.S.: Pay attention that the documentation hasn't been updated yet  for React Router v6, the only info about the new version can be found in GitHub on releases tub. So if something isn't working even though you do the same way it's written in docs, just downgrade the version of your React Router package to the latest 5th one

Reply:
resolved